src/components/carousel/index.test.jsx

```diff
@@ -6,11 +6,11 @@ import Carousel from ".";
 
 // define full screen like in browsers that support full screen api
 Object.defineProperty(global.document, "fullscreenEnabled", {
-	value: true,
```

Comment:
some of these trailig commas were already here, i just let them get removed so the file overall would be more consistent since we dont seem to use them elsewhere in the repo